tool/tele/cli/register.go

```diff
@@ -33,38 +33,38 @@ func RegisterCommands(app *kingpin.Application) Application {
+	tele.VersionCmd.Output = common.Format(tele.VersionCmd.Flag("output", "Output format: text or json.").Short('o').Default(string(constants.EncodingText)))
 
-	tele.BuildCmd.CmdClause = app.Command("build", "Build an application installer")
-	tele.BuildCmd.ManifestPath = tele.BuildCmd.Arg("manifest-path", fmt.Sprintf("Path to the application manifest file, must be %q", defaults.ManifestFileName)).Default(defaults.ManifestFileName).String()
-	tele.BuildCmd.OutFile = tele.BuildCmd.Flag("output", "Name of the generated tarball, defaults to <dirname>.tar.gz where <dirname> is the name of the directory where app manifest is located").Short('o').String()
-	tele.BuildCmd.Overwrite = tele.BuildCmd.Flag("overwrite", "Overwrite the existing tarball").Short('f').Bool()
-	tele.BuildCmd.Repository = tele.BuildCmd.Flag("repository", "Optional address of Gravity Hub to download dependencies from").Hidden().String()
-	tele.BuildCmd.Name = tele.BuildCmd.Flag("name", "Optional application name, overrides the one specified in the manifest file").Hidden().String()
-	tele.BuildCmd.Version = tele.BuildCmd.Flag("version", "Optional application version, overrides the one specified in the manifest file").Hidden().String()
-	tele.BuildCmd.VendorPatterns = tele.BuildCmd.Flag("glob", "File pattern to search for container image references").Default(defaults.VendorPattern).Hidden().Strings()
-	tele.BuildCmd.VendorIgnorePatterns = tele.BuildCmd.Flag("ignore", "Ignore files matching this regular expression when searching for container references").Hidden().Strings()
-	tele.BuildCmd.SetImages = loc.ImagesSlice(tele.BuildCmd.Flag("set-image", "Rewrite docker image versions in the application resource files during vendoring, e.g. 'postgres:9.3.4' will rewrite all images with name 'postgres' to 'postgres:9.3.4'").Hidden())
-	tele.BuildCmd.SetDeps = loc.LocatorSlice(tele.BuildCmd.Flag("set-dep", "Rewrite dependencies section in the application manifest file during vendoring, e.g. 'gravitational.io/site-app:0.0.39' will overwrite dependency to 'gravitational.io/site-app:0.0.39'").Hidden())
-	tele.BuildCmd.SkipVersionCheck = tele.BuildCmd.Flag("skip-version-check", "Skip version compatibility check").Hidden().Bool()
-	tele.BuildCmd.Parallel = tele.BuildCmd.Flag("parallel", "Specifies the number of concurrent tasks. If < 0, the number of tasks is not restricted, if unspecified, then tasks are capped at the number of logical CPU cores").Int()
-	tele.BuildCmd.Quiet = tele.BuildCmd.Flag("quiet", "Suppress any extra output to stdout").Short('q').Bool()
+	tele.BuildCmd.CmdClause = app.Command("build", "Cluster and application image build tool.")
+	tele.BuildCmd.ManifestPath = tele.BuildCmd.Arg("path", fmt.Sprintf("Path to the cluster image manifest file (must be named %q), or unpacked Helm chart to build an application image out of.", defaults.ManifestFileName)).Default(defaults.ManifestFileName).String()
```

Comment:
Why does a helm chart need to be unpacked? Can it simply mention a `helm chart `?

Reply:
Right now the chart has to be unpacked so I thought mentioning it would be useful. We could (and probably should) implement so tele can build app image from an unpacked chart too.